### PR TITLE
phpstan: fixed unknown types

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

...

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)
 - [x] Add yourself to contributors list

### CODE REVIEW

**Review by fballiano — DISMISSED** [review]

File: app/code/core/Mage/Dataflow/Model/Batch.php [hunk 1]
```diff
@@ -151,7 +151,7 @@ public function getBatchExportModel()
     /**
      * Retrieve Batch import model
      *
-     * @return Mage_Dataflow_Model_Import_Export
+     * @return Mage_Dataflow_Model_Import
```
Comment (kiatng) [inline app/code/core/Mage/Dataflow/Model/Batch.php:154]: See line 159,`$object = Mage::getModel('dataflow/batch_import');` it should be:
```suggestion
     * @return Mage_Dataflow_Model_Batch_Import
```

File: app/code/core/Mage/Dataflow/Model/Convert/Parser/Abstract.php [hunk 2]
```diff
@@ -87,7 +87,7 @@ public function getBatchExportModel()
     /**
      * Retrieve Batch import model
      *
-     * @return Mage_Dataflow_Model_Import_Export
+     * @return Mage_Dataflow_Model_Import
```
Comment (kiatng) [inline app/code/core/Mage/Dataflow/Model/Convert/Parser/Abstract.php:90]: See line 95,`$object = Mage::getModel('dataflow/batch_import');` it should be:
```suggestion
     * @return Mage_Dataflow_Model_Batch_Import
```

File: app/code/core/Mage/Install/Model/Installer.php [hunk 3]
```diff
@@ -254,7 +254,7 @@ public function createAdministrator($data)
      * Returns TRUE or array of error messages.
      *
      * @param string $key
-     * @return unknown_type
+     * @return array|true
```
Comment (kiatng) [inline app/code/core/Mage/Install/Model/Installer.php:257]: Is this better?
```suggestion
     * @return string[]|true
```

File: app/code/core/Mage/Dataflow/Model/Batch.php (hunk 1 above)
Comment (sreichel) [reply app/code/core/Mage/Dataflow/Model/Batch.php]: thanks

**Review by kiatng — APPROVED** [review]

**Review by ADDISON74 — APPROVED** [review]

**Pull request merged**